File: Algoritmos e Logica de Programacao/Python/Estrutura Condicional/ProblemaMultiplos.py
'''Problema "multiplos" (adaptado de URI 1044)
Fazer um programa para ler dois números inteiros, e dizer se um número é múltiplo do outro. Os
números podem ser digitados em qualquer ordem.'''

x: int; y: int;

print("Digite dois numeros inteiros: ")
x = int(input())
y = int(input())

if x % y == 0 or y % x == 0:
    print("Sao multiplos")
else:
    print("Nao sao multiplos")

File: Algoritmos e Logica de Programacao/C++/Estrutura Condicional/ProblemaMultiplos.cpp
/*
Problema "multiplos" (adaptado de URI 1044)
Fazer um programa para ler dois números inteiros, e dizer se um número é múltiplo do outro. Os
números podem ser digitados em qualquer ordem.
*/


#include <iostream>
#include <iomanip>

using namespace std;

int main(){

	int a, b;
	cout << "Digite dois numeros inteiros: " << endl;
	cin >> a;
	cin >> b;
	
	if((a % b == 0)||(b % a == 0)){
		cout << "Sao multiplos ";
	}else{
		cout << "Nao sao multiplos ";
	}
    return 0;
}
